app/src/main/res/navigation/nav_graph.xml

```diff
@@ -16,9 +16,9 @@
     </fragment>
     <fragment
         android:id="@+id/mainscreen_placeholder"
```

Comment:
mainscreen_placeholder id is not valid anymore, should be related to launch screen